test/libdevcore/rlp.cpp

```diff
@@ -40,8 +40,86 @@ namespace js = json_spirit;
 namespace dev
 {
 	namespace test
-	{
-		static void buildRLP(js::mValue& _v, RLPStream& _rlp)
+	{	
+		void buildRLP(js::mValue& _v, RLPStream& _rlp);
+		void checkRLPAgainstJson(js::mValue& v, RLP& u);
+
+		void doRlpTests(json_spirit::mValue& v, bool _fillin)
```

Comment:
By creating shorter functions that are called from this function. Please also explain the meaning of the `_fillin` parameter.